core.d.ts

```diff
@@ -141,6 +141,8 @@ declare namespace core {
 		| '3mf'
 		| 'zst'
 		| 'jxl'
+		| 'ntf'
+		| 'nitf'
```

Comment:
We don't add extensions that aren't returned.

Reply:
removed, 
however, I left a note about 'nitf' in package.json (in keywords section), b/c we have files with both extensions, and for both  above detecting code works fine. Let me know if that is OK with you.